modules/changelog.sh

```diff
@@ -1,4 +1,11 @@
 function changelog {
+if [ -d "tests" ]; then
+  cp -f "/usr/lib/node_modules/@treehouses/cli/CHANGELOG.md" . 2>/dev/null || true
+  rm -f "/usr/lib/node_modules/@treehouses/cli/CHANGELOG.md"
+else
+  cp -f "/usr/lib/node_modules/@treehouses/cli/CHANGELOG.md" ../. 2>/dev/null || true
+  rm -f "/usr/lib/node_modules/@treehouses/cli/CHANGELOG.md"
+fi
```

Comment:
You can delete this. This belongs in the changelog.bats

Reply:
> You can delete this. This belongs in the changelog.bats

I deleted the code you mentioned.